mmv1/products/sql/api.yaml

```diff
@@ -726,6 +726,30 @@ objects:
             description: |
               The externally accessible port for the source database server.
               Defaults to 3306.
+          - !ruby/object:Api::Type::String
+            name: 'username'
+            description: |
+              The username for connecting to the on-premises instance.
+          - !ruby/object:Api::Type::String
+            name: 'password'
+            description: |
+              The password for connecting to the on-premises instance.
+          - !ruby/object:Api::Type::String
+            name: 'ca_certificate'
+            description: |
+              PEM representation of the trusted CA's x509 certificate.
+          - !ruby/object:Api::Type::String
+            name: 'client_certificate'
+            description: |
+              PEM representation of the replica's x509 certificate.
+          - !ruby/object:Api::Type::String
+            name: 'client_key'
+            description: |
+              PEM representation of the replica's private key. The corresponsing public key is encoded in the client's certificate.
+          - !ruby/object:Api::Type::String
+            name: 'dump_file_path'
```

Comment:
```suggestion
            name: 'dumpFilePath'
```

Reply:
Ah interesting! Sorry I missed this one!